File: sentimental-mario-less/mario.py
from cs50 import get_int


# Prompt the user with get_int for height
# Re-prompt until user provides an integer between 1 and 9
def get_height():
    while True:
        size = get_int("Height: ")
        if size in range(1, 9):
            break
    return size


# Print the desired height of pyramid
def main():
    height = get_height()
    for i in range(height):
        for j in range(height - i - 1):
            print(" ", end="")
        for j in range(i + 1):
            print("#", end="")
        print("")


main()

File: mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //Prompt user for height
    int size;
    do
    {
        size = get_int("Height: ");
    }
    while (size < 1 || size > 8);

    //Height of pyramid loop
    for (int i = 0; i < size; i++)
    {
        //Adding spaces
        for (int j = size - 1; j > i; j--)
        {
            printf(" ");
        }

        //Adding hashes
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}